src/main/ruby/truffleruby/core/file.rb

```diff
@@ -37,9 +37,12 @@
 class File < IO
   include Enumerable
 
-  class FileError < Exception; end # rubocop:disable Lint/InheritException
+  class FileError < RuntimeError; end
```

Comment:
Why did you change this?
Such change has nothing to do with this PR AFAIK.
But as @andrykonchin says below, you could remove those 4 classes in a separate commit in this PR as a cleanup if you like, or undo these changes.

Reply:
It was complained about by rubocop, and since rubocop was being run by the pre-commit hook, I couldn't commit without fixing it. I don't know why the hook let me commit before, and I don't remember pulling any new changes to the lint rules since I started the branch.

Anyway, them being not needed simplifies things a lot, I will delete them in a seperate commit and if the CI didn't fail then that's all good.

Reply:
Mmh, `jt rubocop` should pick the right version and config, and it does work for other people so I suspect something was broken in your dev env or so.